src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        displayWelcomeMsg();
+
+        Task[] array = new Task[100];
+        String line;
+        Scanner in = new Scanner(System.in);
+        boolean looper = true;
+
+        while(looper) {
```

Comment:
perhaps can put the while loop that handles commands into a function instead of main